activesupport/lib/active_support/core_ext/enumerable.rb

```diff
@@ -60,6 +60,19 @@ def many?
   def exclude?(object)
     !include?(object)
   end
+
+  # Returns a copy of the enumerable without the specified elements.
+  #
+  #   people = ["David", "Rafael", "Aaron", "Todd"]
+  #   people.without "Aaron", "Todd"
+  #     => ["David", "Rafael"]
+  def without(*elements)
+    if is_a? Array
+      self - elements
```

Comment:
:+1: to overwriting in an Array core extension. We do that with other methods like blank? etc.